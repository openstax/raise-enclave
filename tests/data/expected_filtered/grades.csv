assessment_id,user_uuid,course_id,grade_percentage,time_submitted
1,d35a986b-a7ad-4c61-8e76-205eccaf9a06,3,80.0,1661272186
