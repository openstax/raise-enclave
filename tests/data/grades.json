{
  "usergrades": [
    {
      "courseid": 2,
      "courseidnumber": "",
      "gradeitems": [
        {
          "categoryid": 2,
          "cmid": 17,
          "feedback": "",
          "feedbackformat": 0,
          "gradedategraded": 1661272078,
          "gradedatesubmitted": 1661272078,
          "gradeformatted": "0.00",
          "gradehiddenbydate": false,
          "gradeishidden": false,
          "gradeislocked": false,
          "gradeisoverridden": false,
          "grademax": 10,
          "grademin": 0,
          "gradeneedsupdate": false,
          "graderaw": 0,
          "id": 16,
          "idnumber": null,
          "iteminstance": 1,
          "itemmodule": "quiz",
          "itemname": "Unit 1, Section A Quiz",
          "itemnumber": 0,
          "itemtype": "mod",
          "locked": false,
          "outcomeid": null,
          "percentageformatted": "0.00 %",
          "rangeformatted": "0&ndash;10",
          "scaleid": null,
          "weightformatted": "11.11 %",
          "weightraw": 0.11111
        },
        {
          "categoryid": 2,
          "cmid": 31,
          "feedback": "",
          "feedbackformat": 0,
          "gradedategraded": 1661272186,
          "gradedatesubmitted": 1661272186,
          "gradeformatted": "8.00",
          "gradehiddenbydate": false,
          "gradeishidden": false,
          "gradeislocked": false,
          "gradeisoverridden": false,
          "grademax": 10,
          "grademin": 0,
          "gradeneedsupdate": false,
          "graderaw": 8,
          "id": 29,
          "idnumber": null,
          "iteminstance": 2,
          "itemmodule": "quiz",
          "itemname": "Unit 1, Section B Quiz",
          "itemnumber": 0,
          "itemtype": "mod",
          "locked": false,
          "outcomeid": null,
          "percentageformatted": "80.00 %",
          "rangeformatted": "0&ndash;10",
          "scaleid": null,
          "weightformatted": "11.11 %",
          "weightraw": 0.11111
        }
      ],
      "maxdepth": 2,
      "userfullname": "Marguerite Bell",
      "userid": 9,
      "useridnumber": ""
    },
    {
      "courseid": 2,
      "courseidnumber": "",
      "gradeitems": [
        {
          "categoryid": 2,
          "cmid": 17,
          "feedback": "",
          "feedbackformat": 0,
          "gradedategraded": 1661273642,
          "gradedatesubmitted": 1661273642,
          "gradeformatted": "0.00",
          "gradehiddenbydate": false,
          "gradeishidden": false,
          "gradeislocked": false,
          "gradeisoverridden": false,
          "grademax": 10,
          "grademin": 0,
          "gradeneedsupdate": false,
          "graderaw": 0,
          "id": 16,
          "idnumber": null,
          "iteminstance": 1,
          "itemmodule": "quiz",
          "itemname": "Unit 1, Section A Quiz",
          "itemnumber": 0,
          "itemtype": "mod",
          "locked": false,
          "outcomeid": null,
          "percentageformatted": "0.00 %",
          "rangeformatted": "0&ndash;10",
          "scaleid": null,
          "weightformatted": "11.11 %",
          "weightraw": 0.11111
        },
        {
          "categoryid": 2,
          "cmid": 31,
          "feedback": "",
          "feedbackformat": 0,
          "gradedategraded": 1661273664,
          "gradedatesubmitted": 1661273664,
          "gradeformatted": "6.00",
          "gradehiddenbydate": false,
          "gradeishidden": false,
          "gradeislocked": false,
          "gradeisoverridden": false,
          "grademax": 10,
          "grademin": 0,
          "gradeneedsupdate": false,
          "graderaw": 6,
          "id": 29,
          "idnumber": null,
          "iteminstance": 2,
          "itemmodule": "quiz",
          "itemname": "Unit 1, Section B Quiz",
          "itemnumber": 0,
          "itemtype": "mod",
          "locked": false,
          "outcomeid": null,
          "percentageformatted": "60.00 %",
          "rangeformatted": "0&ndash;10",
          "scaleid": null,
          "weightformatted": "11.11 %",
          "weightraw": 0.11111
        }
      ],
      "maxdepth": 2,
      "userfullname": "David Tucci",
      "userid": 7,
      "useridnumber": ""
    },
    {
      "courseid": 2,
      "courseidnumber": "",
      "gradeitems": [
        {
          "categoryid": 2,
          "cmid": 17,
          "feedback": "",
          "feedbackformat": 0,
          "gradedategraded": null,
          "gradedatesubmitted": null,
          "gradeformatted": "-",
          "gradehiddenbydate": false,
          "gradeishidden": false,
          "gradeislocked": false,
          "gradeisoverridden": false,
          "grademax": 10,
          "grademin": 0,
          "gradeneedsupdate": false,
          "graderaw": null,
          "id": 16,
          "idnumber": null,
          "iteminstance": 1,
          "itemmodule": "quiz",
          "itemname": "Unit 1, Section A Quiz",
          "itemnumber": 0,
          "itemtype": "mod",
          "locked": false,
          "outcomeid": null,
          "percentageformatted": "-",
          "rangeformatted": "0&ndash;10",
          "scaleid": null,
          "status": "novalue",
          "weightformatted": "0.00 %",
          "weightraw": 0
        },
        {
          "categoryid": 2,
          "cmid": 31,
          "feedback": "",
          "feedbackformat": 0,
          "gradedategraded": 1661273350,
          "gradedatesubmitted": 1661273350,
          "gradeformatted": "6.00",
          "gradehiddenbydate": false,
          "gradeishidden": false,
          "gradeislocked": false,
          "gradeisoverridden": false,
          "grademax": 10,
          "grademin": 0,
          "gradeneedsupdate": false,
          "graderaw": 6,
          "id": 29,
          "idnumber": null,
          "iteminstance": 2,
          "itemmodule": "quiz",
          "itemname": "Unit 1, Section B Quiz",
          "itemnumber": 0,
          "itemtype": "mod",
          "locked": false,
          "outcomeid": null,
          "percentageformatted": "60.00 %",
          "rangeformatted": "0&ndash;10",
          "scaleid": null,
          "weightformatted": "20.00 %",
          "weightraw": 0.2
        }
      ],
      "maxdepth": 2,
      "userfullname": "James Stewart",
      "userid": 8,
      "useridnumber": ""
    },
    {
      "courseid": 2,
      "courseidnumber": "",
      "gradeitems": [
        {
          "categoryid": 2,
          "cmid": 17,
          "feedback": "",
          "feedbackformat": 0,
          "gradedategraded": null,
          "gradedatesubmitted": null,
          "gradeformatted": "-",
          "gradehiddenbydate": false,
          "gradeishidden": false,
          "gradeislocked": false,
          "gradeisoverridden": false,
          "grademax": 10,
          "grademin": 0,
          "gradeneedsupdate": false,
          "graderaw": null,
          "id": 16,
          "idnumber": null,
          "iteminstance": 1,
          "itemmodule": "quiz",
          "itemname": "Unit 1, Section A Quiz",
          "itemnumber": 0,
          "itemtype": "mod",
          "locked": false,
          "outcomeid": null,
          "percentageformatted": "-",
          "rangeformatted": "0&ndash;10",
          "scaleid": null,
          "status": "novalue",
          "weightformatted": "0.00 %",
          "weightraw": 0
        },
        {
          "categoryid": 2,
          "cmid": 31,
          "feedback": "",
          "feedbackformat": 0,
          "gradedategraded": null,
          "gradedatesubmitted": null,
          "gradeformatted": "-",
          "gradehiddenbydate": false,
          "gradeishidden": false,
          "gradeislocked": false,
          "gradeisoverridden": false,
          "grademax": 10,
          "grademin": 0,
          "gradeneedsupdate": false,
          "graderaw": null,
          "id": 29,
          "idnumber": null,
          "iteminstance": 2,
          "itemmodule": "quiz",
          "itemname": "Unit 1, Section B Quiz",
          "itemnumber": 0,
          "itemtype": "mod",
          "locked": false,
          "outcomeid": null,
          "percentageformatted": "-",
          "rangeformatted": "0&ndash;10",
          "scaleid": null,
          "status": "novalue",
          "weightformatted": "0.00 %",
          "weightraw": 0
        }
      ],
      "maxdepth": 2,
      "userfullname": "Deann James",
      "userid": 11,
      "useridnumber": ""
    }
  ],
  "warnings": []
}