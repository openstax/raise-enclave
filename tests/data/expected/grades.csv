assessment_id,user_uuid,course_id,grade_percentage,time_submitted
0,7080c78d-298b-40ba-a68d-55d6a93b00fb,2,0.0,1661272078
1,7080c78d-298b-40ba-a68d-55d6a93b00fb,2,80.0,1661272186
0,7080c78d-298b-40ba-a68d-55d6a93b00fb,2,0.0,1661273642
1,7080c78d-298b-40ba-a68d-55d6a93b00fb,2,60.0,1661273664
1,7080c78d-298b-40ba-a68d-55d6a93b00fb,2,60.0,1661273350
